Kickstarter-iOS/AppDelegate.swift

```diff
@@ -144,22 +147,28 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
       .observeForUI()
       .observeValues(UIApplication.shared.unregisterForRemoteNotifications)
 
-    self.viewModel.outputs.configureHockey
+    self.viewModel.outputs.configureAppCenterWithData
       .observeForUI()
-      .observeValues { [weak self] data in
-        guard let _self = self else { return }
-        let manager = BITHockeyManager.shared()
-        manager.delegate = _self
-        manager.configure(withIdentifier: data.appIdentifier)
-        manager.crashManager.crashManagerStatus = .disabled
-        manager.isUpdateManagerDisabled = data.disableUpdates
-        manager.userID = data.userId
-        manager.userName = data.userName
-        manager.start()
-        manager.authenticator.authenticateInstallation()
+      .observeValues { data in
+        let customProperties = MSCustomProperties()
+        customProperties.setString(data.userName, forKey: "userName")
```

Comment:
Added this because there doesn't appear to be a similar `userName` property like `BITHockeyManager` had before.